src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7312,6 +7312,7 @@ public sealed partial class StringBuilder : System.Runtime.Serialization.ISerial
         public void CopyTo(int sourceIndex, System.Span<char> destination, int count) { }
         public int EnsureCapacity(int capacity) { throw null; }
         public bool Equals(System.Text.StringBuilder sb) { throw null; }
+        public bool Equals(System.ReadOnlySpan<char> value) { throw null; }
```

Comment:
In the API proposal (#25846) the name of the parameter is `s` but here it is `value`. What should it be?

Reply:
@danmosemsft do we need to keep it ```s``` only or ```value``` is fine ?

Reply:
In general, we should follow the approved API shape & names, unless we want to bring it back to API review to make changes.

In this particular case:
`s` seems to be consistent with `sb` above. Across the source code we use both approaches, so consistency in type is preferred.
However, the name `s` was there originally for `string` type, which does not fit well the final type we approved `ReadOnlySpan<char>`.
Personally, I'd be fine using `value` - maybe we should run it by other API reviewers? (e.g. over email).

Reply:
who are the reviewers for this API ?